app/assets/javascripts/discourse/app/components/plugin-outlet.hbs

```diff
@@ -38,7 +42,11 @@
         @class={{c.classicClassNames}}
         @tagName={{or @connectorTagName ""}}
```

Comment:
Curiosity, would just `@tagName={{@connectorTagName}}` just achieve the same here or do we need actual empty string?